doc/languages-frameworks/go.section.md

```diff
@@ -112,16 +112,6 @@ done
 
 Both `buildGoModule` and `buildGoPackage` can be tweaked to behave slightly differently, if the following attributes are used:
 
-### `buildFlagsArray` and `buildFlags`: {#ex-goBuildFlags-noarray}
-
-These attributes set build flags supported by `go build`. We recommend using `buildFlagsArray`.
```

Comment:
Don't we still need `buildFlagsArray` to set a version? Maybe we just need to change the example here.

Reply:
Or is this covered by `-ldflags`?

Reply:
It's covered by `ldflags`.

Reply:
Ah. There is already documentation for that!